chainer/functions/array/tile.py

```diff
@@ -66,13 +66,76 @@ def backward(self, inputs, grads):
 def tile(x, reps):
     """Construct an array by tiling a given array.
 
+
     Args:
-        x (chainer.Variable or :class:`numpy.ndarray` or cupy.ndarray):
-            Input data.
-        reps (int or tuple of ints): The number of times for each axis with
-            which x is replicated.
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variable. Let the length of ``reps`` be ``d`` and
```

Comment:
I think it is better not to use let for conditioning `x.ndim < d`. So, how about "Let the length of `reps` be `d`. If `x.ndim < d`, " ...?